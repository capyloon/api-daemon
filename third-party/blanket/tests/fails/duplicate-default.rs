extern crate blanket;
use blanket::blanket;

#[blanket(default = "default", default = "other")]
pub trait MyTrait {}

pub fn main() {}
