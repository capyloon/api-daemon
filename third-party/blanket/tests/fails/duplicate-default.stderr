error: duplicate default module given
 --> $DIR/duplicate-default.rs:4:42
  |
4 | #[blanket(default = "default", default = "other")]
  |                                          ^^^^^^^
